main.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/hoshsadiq/godotenv"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+type sliceString []string
+
+func (i *sliceString) String() string {
+	if i == nil {
+		return ""
+	}
+	return strings.Join(*i, ";\n") + ";\n"
+}
+
+func (i *sliceString) Set(value string) error {
+	vars, err := godotenv.Unmarshal(value)
+	if err != nil {
+		return err
+	}
+
+	for k, v := range vars {
+		*i = append(*i, fmt.Sprintf("%s=%s", k, v))
+	}
+	return nil
+}
+
+type DeployOptions struct {
+	branch    string
+	file      string
+	logLevel  string
+	name      string
+	scope     string
+	timeout   time.Duration
+	variables sliceString
+	wait      bool
+	comment   string
+
+	ci ciInfo
+}
+
+func exitIfErr(err error) {
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(2)
+	}
+}
+
+/*
+todo: the follow code is left:
+
+	if [ ! -z "$OKTETO_CA_CERT" ]; then
+	   echo "Custom certificate is provided"
+	   echo "$OKTETO_CA_CERT" > /usr/local/share/ca-certificates/okteto_ca_cert.crt
+	   update-ca-certificates
+	fi
```

Comment:
I'm unsure where this is used, so haven't made the relevant changes yet.

Reply:
This is to support the scenario where Okteto is using a private CA or a self-signed certificate